apps/snitch_core/lib/core/data/schema/option_type.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Snitch.Data.Schema.OptionType do
+  @moduledoc false
+  use Snitch.Data.Schema
+
+  alias Snitch.Data.Schema.OptionValue
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_option_types" do
+    field(:name, :string)
+    field(:display_name, :string)
+    field(:position, :integer)
+    timestamps()
+
+    has_many(:option_values, OptionValue)
+  end
+
+  @create_params ~w(name display_name)a
+
+  @doc """
+  Returns a changeset to create new OptionType
+  """
+  @spec create_changeset(t, map) :: Ecto.Changeset.t()
+  def create_changeset(model, params) do
+    common_changeset(model, params)
+  end
+
+  @doc """
+  Returns a changeset to update a OptionType
+  """
+  @spec update_changeset(t, map) :: Ecto.Changeset.t()
+  def update_changeset(model, params) do
+    common_changeset(model, params)
+  end
+
+  defp common_changeset(model, params) do
+    model
+    |> cast(params, @create_params)
```

Comment:
The `:position` is never casted. The utility of that field is not documented anywhere.

Reply:
Position is used to set the order in which the option type will be shown on UI.

Will remove this as the UI drag drop for list item is not implemented yet.